public/assets/.sprockets-manifest-d0444265d8bed6e98efa1a64f58d5fa1.json

```diff
@@ -0,0 +1 @@
f478c1d29aa1c08b764b603b90","integrity":"sha256-z9Bf+6GjOGPv5kpA0fL9T8zA5PR4wdKaocCLdktgO5A="},"logo-1e7f10045d99ba79552dfff16c010799b6f7dae032acadc58b910e17417dab43.png":{"logical_path":"logo.png","mtime":"2017-08-15T20:16:27+07:00","size":31215,"digest":"1e7f10045d99ba79552dfff16c010799b6f7dae032acadc58b910e17417dab43","integrity":"sha256-Hn8QBF2ZunlVLf/xbAEHmbb32uAyrK3Fi5EOF0F9q0M="},"up-clear-3ff04d45ee21ef715d898f3c9053678597dd3e932563ba34940bcd2662b371d4.png":{"logical_path":"up-clear.png","mtime":"2017-08-15T20:16:27+07:00","size":1518,"digest":"3ff04d45ee21ef715d898f3c9053678597dd3e932563ba34940bcd2662b371d4","integrity":"sha256-P/BNRe4h73FdiY88kFNnhZfdPpMlY7o0lAvNJmKzcdQ="},"up-fill-87a690ddc4dc2b0cb699c3ab82cd8f0fc2f09ee98497a99f5dd46bdf3087bdb9.png":{"logical_path":"up-fill.png","mtime":"2017-08-15T20:16:27+07:00","size":1190,"digest":"87a690ddc4dc2b0cb699c3ab82cd8f0fc2f09ee98497a99f5dd46bdf3087bdb9","integrity":"sha256-h6aQ3cTcKwy2mcOrgs2PD8LwnumEl6mfXdRr3zCHvbk="},"application-e19e7afeb43915e2165e0b0d4f927b842f594b39ea449a76251b4dd068dd7e79.js":{"logical_path":"application.js","mtime":"2017-08-16T15:40:20+07:00","size":354722,"digest":"e19e7afeb43915e2165e0b0d4f927b842f594b39ea449a76251b4dd068dd7e79","integrity":"sha256-4Z56/rQ5FeIWXgsNT5J7hC9ZSznqRJp2JRtN0Gjdfnk="},"application-fbf25f87b816945058df04f4e29fe6317f0e0da6028c95c47944560f74bdf5a2.css":{"logical_path":"application.css","mtime":"2017-08-16T15:40:20+07:00","size":5920,"digest":"fbf25f87b816945058df04f4e29fe6317f0e0da6028c95c47944560f74bdf5a2","integrity":"sha256-+/Jfh7gWlFBY3wT04p/mMX8ODaYCjJXEeURWD3S99aI="},"application-0919e3b4bb6f47e19f6fe43bf31dc8f43fb841ba2de0e396ac9180b717533b6e.js":{"logical_path":"application.js","mtime":"2017-08-16T15:40:20+07:00","size":129664,"digest":"0919e3b4bb6f47e19f6fe43bf31dc8f43fb841ba2de0e396ac9180b717533b6e","integrity":"sha256-CRnjtLtvR+Gfb+Q78x3I9D+4Qbot4OOWrJGAtxdTO24="},"application-358bc2202961a3d3cf7b32db96a61f332c69dffcc530888f8fe916809e14d9fa.css":{"logical_path":"application.css","mtime":"2017-08-16T15:40:20+07:00","size":2698,"digest":"358bc2202961a3d3cf7b32db96a61f332c69dffcc530888f8fe916809e14d9fa","integrity":"sha256-NYvCIClho9PPezLblqYfMyxp3/zFMIiPj+kWgJ4U2fo="}},"assets":{"down-clear.png":"down-clear-137f65240ec6ef02feaf98d86122fb5308b8977161a37670ab6ef868d74de37e.png","down-fill.png":"down-fill-49126c68b98ed5951765cf7d1ad82779fbe577e58e26aec9ca7152f792b5abce.png","favicon.png":"favicon-cfd05ffba1a33863efe64a40d1f2fd4fccc0e4f478c1d29aa1c08b764b603b90.png","logo.png":"logo-1e7f10045d99ba79552dfff16c010799b6f7dae032acadc58b910e17417dab43.png","up-clear.png":"up-clear-3ff04d45ee21ef715d898f3c9053678597dd3e932563ba34940bcd2662b371d4.png","up-fill.png":"up-fill-87a690ddc4dc2b0cb699c3ab82cd8f0fc2f09ee98497a99f5dd46bdf3087bdb9.png","application.js":"application-0919e3b4bb6f47e19f6fe43bf31dc8f43fb841ba2de0e396ac9180b717533b6e.js","application.css":"application-358bc2202961a3d3cf7b32db96a61f332c69dffcc530888f8fe916809e14d9fa.css"}}
```

Comment:
Vendored/minified file -- it might (might!) be helpful in prod, but it's never helpful in a development repository.